source/elements/oneDAL/source/common_interface/header_files_and_namespaces.rst

```diff
@@ -80,6 +80,14 @@ oneDAL functionality is represented with a system of C++ namespaces described be
        should be substituted with its name, for example, ``oneapi::dal::kmeans``
        or ``oneapi::dal::knn``.
 
+   * - ``oneapi::dal::%DATA_SOURCE%``
+
+     - The namespace of the data source.
+       All classes and structures related to that data source shall be defined within a particular namespace.
+
+       To define a specific data source, the string %DATA_SOURCE% should be substituted with its name,
+       for example, ``oneapi::dal::csv``.
```

Comment:
Is this namespace related only to data source, or to all file-related functionality in oneDAL?

Reply:
It can potentially contain some functionality related to `data-sink` (write table to csv), but I can't see anything else beyond that.